docs/specs/oidc.md

```diff
@@ -184,7 +190,13 @@ Unknown parameters are ignored, and invalid parameters are rejected. However, if
 
 ### acr_values
 
-Unsupported yet.
+`acr_values` determines how the authentication is performed when both a valid session and `id_token_hint` are present.
+
+The value is an URL `https://authgear.com/acr` with the following query parameters.
+
+- `primary_authenticator=password`: Indicate that the primary authenticator was password. Other values are possible.
+- `secondary_authenticator=totp`: Indicate that the secondary authenticator was TOTP. Other values are possible.
+- `reauthenticate=primary_authenticator`: Require that one of primary authenticators of the users to be used to reauthenticate the user.
```

Comment:
Can we list out the values that we support for re-authentication? Or we will only support `primary_authenticator` in this stage?